src/components/tl/mlx5/mcast/p2p/ucc_tl_mlx5_mcast_p2p.c

```diff
@@ -0,0 +1,164 @@
+/**
+ * Copyright (c) 2022-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+ *
+ * See file LICENSE for terms.
+ */
+
+#include "ucc_tl_mlx5_mcast_p2p.h"
+
+static inline void ucc_tl_mlx5_mcast_p2p_completion_cb(void* context)
+{
+    ucc_tl_mlx5_mcast_p2p_completion_obj_t *obj =
+        (ucc_tl_mlx5_mcast_p2p_completion_obj_t *)context;
+
+    ucc_assert(obj != NULL && obj->compl_cb != NULL);
+
+    obj->compl_cb(obj);
+
+    ucc_assert(obj->req != NULL);
+
+    ucc_collective_finalize(obj->req);
+}
+
+void ucc_tl_mlx5_mcast_completion_cb(void* context, ucc_status_t status) //NOLINT
+{
+    ucc_tl_mlx5_mcast_p2p_completion_cb(context);
+}
+
+static inline ucc_status_t
+              ucc_tl_mlx5_mcast_do_p2p_bcast_nb(void                *buf,
```

Comment:
alignment is not good, see here for reference https://github.com/openucx/ucc/blob/master/docs/codestyle.md